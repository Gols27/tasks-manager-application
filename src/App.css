.main-container {
  min-height: 70vh;
  padding-bottom:  15vh;
  display: flex;
}