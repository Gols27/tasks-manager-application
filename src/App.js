import React from "react";
import Landing from "./landing";
import Home from "./home";
import Header from "./common/header";
import Footer from "./common/footer";
import Login from "./login";
import "bootstrap/dist/css/bootstrap.min.css";

import { BrowserRouter as Router, Switch, Route } from "react-router-dom";

import "./App.css";

function App() {
  return (
    <Router>
      <Header />
      <div className="main-container">
        <Switch>
          <Route exact path="/">
            <Landing />
          </Route>
          <Route path="/home">
            <Home />
          </Route>
          <Route path="/login">
            <Login />
          </Route>

          {/* <Route component={Notfound} /> */}
        </Switch>
      </div>
      <Footer />
    </Router>
  );
}

export default App;
