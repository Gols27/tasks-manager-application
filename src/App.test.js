import React from 'react';
import App from './App';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';


it('renders without crashing', () => {
  const wrapper = shallow(<App />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
