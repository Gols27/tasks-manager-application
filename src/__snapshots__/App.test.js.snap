// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BrowserRouter>
  <Header />
  <div
    className="main-container"
  >
    <Switch>
      <Route
        exact={true}
        path="/"
      >
        <Landing />
      </Route>
      <Route
        path="/home"
      >
        <Connect(Home) />
      </Route>
      <Route
        path="/login"
      >
        <Connect(Login) />
      </Route>
    </Switch>
  </div>
  <Footer />
</BrowserRouter>
`;
